cognite_toolkit/_cdf_tk/apps/_migrate_app.py

```diff
@@ -99,11 +113,13 @@ def assets(
     ) -> None:
         """Migrate Assets to CogniteAssets."""
         client = EnvironmentVariables.create_from_environment().get_client()
-        cmd = MigrateAssetsCommand()
+        cmd = MigrationCommand()
         cmd.run(
-            lambda: cmd.migrate_assets(
-                client,
-                mapping_file=mapping_file,
+            lambda: cmd.migrate(
+                selected=MigrationCSVFileSelector(mapping_file, resource_type="asset"),
+                data=AssetCentricMigrationIOAdapter(client, AssetIO(client), InstanceIO(client)),
+                mapper=AssetCentricMapper(client),
+                log_dir=log_dir,
```

Comment:
Note this is not exposed to the end user yet. see line 28.

It cannot be until the Migration model is ready (ongoing work https://github.com/cognitedata/toolkit/pull/1987).